torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6454,6 +6477,18 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            # Need to skip out test because one of the overload for mean does not support it
            # TODO(@heitorschueroff) fix this when implementing ReductionInfo
            skips=(SkipInfo('TestCommon', 'test_out'),)),
+    OpInfo('nanmean',
+           ref=lambda x, dim=None, keepdim=False: np.nanmean(x, axis=dim if x.ndim > 0 else None, keepdims=keepdim),
+           dtypes=floating_types_and(torch.float16, torch.bfloat16),
+           # Failing with: Jacobian mismatch for output 0 with respect to input 0,
+           supports_autograd=False,
+           sample_inputs_func=sample_inputs_nan_reduction(supports_multiple_dims=True),
+           # Need to skip out test because one of the overload for mean does not support it
+           # TODO(@heitorschueroff) fix this when implementing ReductionInfo
+           skips=(
+               SkipInfo('TestCommon', 'test_out'),
```

Comment:
Per the above comment, I think it's important we investigate the set_ behavior, make sure we're clear on whether it's OK for out= or not OK and why, and that test_out can detect that behavior if we decide it's not OK -- so I don't think we should skip test_out. Maybe that means waiting for some ReductionInfo PRs